File: Competitor Answers/Güray DAĞ/Soru14.c
#include <stdio.h>
int main(){
int dizi1[3];
int dizi2[3];
int dizi3[3];
for(int i=0;i<3;i++){
    printf("1. dizinin %d. sayisini giriniz:",i+1);
    scanf("%d",&dizi1[i]);
}
for(int i=0;i<3;i++){
    printf("2. dizinin %d. sayisini giriniz:",i+1);
    scanf("%d",&dizi2[i]);
}
for(int i=0;i<3;i++){
    printf("3. dizinin %d. sayisini giriniz:",i+1);
    scanf("%d",&dizi3[i]);
}
int yeniDizi[3][3];
for(int j=0;j<3;j++){
        yeniDizi[0][j] = dizi1[j];
}
for(int j=0;j<3;j++){
        yeniDizi[1][j] = dizi2[j];
}
for(int j=0;j<3;j++){
        yeniDizi[2][j] = dizi3[j];
}
for(int k=0;k<3;k++){
        printf("\n");
    for(int l=0;l<3;l++){
        printf("%d ",yeniDizi[k][l]);
    }
}
printf("\n--------------");
int yeniDizi2[3][3];
for(int i=0;i<3;i++){
        printf("\n");
    for(int j=0;j<3;j++){
        yeniDizi2[i][j]=yeniDizi[j][i];
        printf("%d ",yeniDizi2[i][j]);
    }
}
return 0;
}

File: Competitor Answers/Hüseyin Bora İÇÖZ - Ramazan Semih UĞUR/soru14.py
dizi1 = [0,0,0]
dizi2 = [0,0,0]
dizi3 = [0,0,0]

matris = [[0,0,0],
		[0,0,0],
		[0,0,0]]

transpoze = [[0,0,0],
		[0,0,0],
		[0,0,0]]

for i in range(1,4):
	temp = [0,0,0]
	for j in range(0,3):
		temp[j] = input(str(i)+". dizinin "+ str(j+1) +". elemanini gir.")
	if i == 1:
		dizi1 = temp
	elif i == 2:
		dizi2 = temp
	elif i == 3:
		dizi3 = temp

birlesikDiziler = dizi1 + dizi2 + dizi3

def matrisHalineGetir(dizi):
	sayac=0
	for i in range(0,3):
		for j in range(0,3):
			matris[i][j]=dizi[sayac]
			sayac = sayac + 1

	for i in range(0,3):
		for j in range(0,3):
			transpoze[i][j] = matris[j][i]

	print("Dizilerin matrise donusmus hali: " +str(matris))
	print("Matrisin transpozesi: " +str(transpoze))

matrisHalineGetir(birlesikDiziler)
